CookieMonster.js

```diff
@@ -4164,8 +4174,7 @@ CM.Sim.SellBuildingsForChoEgg = function() {
 	// Get money made by selling all remaining buildings
 	for (var i in CM.Sim.Objects) {
 		var me = CM.Sim.Objects[i];
-		debug = CM.Sim.BuildingSell(Game.Objects[me.name], Game.Objects[i].basePrice, me.amount, Game.Objects[i].free, me.amount);
-		sellTotal += debug;
+		sellTotal += CM.Sim.BuildingSell(Game.Objects[me.name], Game.Objects[i].basePrice, me.amount, Game.Objects[i].free, me.amount);
```

Comment:
Ugh, this should have been in #355 (last minute fix in the sources but I forgot to regenerate the CookieMonster.js), sorry.

Don't know the best way to deal with that, though?